Sources/Time.swift

```diff
@@ -0,0 +1,64 @@
+#if os(Linux)
+    import Glibc
+#else
+    import Darwin.C
+    var mtid = mach_timebase_info_data_t()
+#endif
+
+/// Milliseconds from January 1, 1970.
+public func now() -> Int {
+    #if os(Linux)
+        var ts = timespec()
+        clock_gettime(CLOCK_MONOTONIC, &ts)
+        return Int(ts.tv_sec) * 1000 + Int(ts.tv_nsec) / 1000000
+    #else
+        if mtid.denom == 0 {
+            mach_timebase_info(&mtid)
+        }
+        let ticks = mach_absolute_time()
+        return Int(ticks * UInt64(mtid.numer) / UInt64(mtid.denom) / 1000000)
+    #endif
+}
+
+public extension Int {
```

Comment:
I think it makes sense to exetend Integer type:

```
extension IntegerType // Integer on Swift 3 {
    func fromNow() -> Int {
        return now() + Int(self.toIntMax()) // Might need to be slightly tweaked, from memory
    }
}
```

Then we get everything for free.